_data/accessibility-tests/checkbox.yml

```diff
@@ -0,0 +1,54 @@
+title: Checkbox
+component_status: pass
+test_items:
+# General tests
+  - summary: Visible instructions explain checkbox requirements. 
+    summary_additional: The checkbox has a visible label above it to indicate its purpose. For example, "Select your state."
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 3.3.2
+  - summary: Radio button labels are descriptive, brief and clear.
+    summary_additional: When you view radio buttons in a checkbox, all labels have short descriptions that are easy to understand and act on. 
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.6
+# Keyboard navigation tests
+  - summary: Focus indicator on checkbox is clearly visible.
+    summary_additional: When you use a keyboard to focus on a checkbox group, you will see a visible outline or other clear indication of focus. 
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 3.2.1
+  - summary: Checkbox focus follows logical content order on desktop.
+    summary_additional: When you tab through the checkbox with a keyboard, the focus stays in order of the content as intended. 
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.3
+  - summary: Checkbox focus can be moved and options can be selected. 
```

Comment:
@finekatie 
cc: @alex-hull @amycole501 

**suggestion (non-blocking):** Wondering if we can simplify this one to focus more on the keyboard interaction. Maybe something like this?
```suggestion
  - summary: Users can access and select checkbox items using only a keyboard.
```

Reply:
Yes! I think we can make this change!

Reply:
Resolved in [5808ba8](https://github.com/uswds/uswds-site/pull/2803/commits/5808ba81a1e73b90ad5129c8a440f7a27c6ad4a9)